### PR TITLE
Issue 765 - Ctrl-F doesn't work in Resend

### PR DESCRIPTION
**Pull request by kingthorin**
Added a Find action to the HttpPanel toolbar (activated by Ctrl-F). 
FindDialog added a constructor that accepts a parent window.

Fixes zaproxy/zaproxy#765

### CODE REVIEW

File: src/org/zaproxy/zap/extension/httppanel/HttpPanel.java [hunk 1]
```diff
@@ -91,6 +101,7 @@
     private JPanel moreOptionsComponent;
     private JToolBar toolBarComponents;
     private JToolBar toolBarMoreOptions;
+    private static Action findAction = null;
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/httppanel/HttpPanel.java]: Could be moved to the same place as the other static variables (null initialisation could be removed).

File: src/org/zaproxy/zap/extension/httppanel/HttpPanel.java [hunk 2]
```diff
@@ -187,6 +200,24 @@ private JPanel getPanelHeader() {
         return panelHeader;
     }
     
+    private Action getFindAction() {
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/httppanel/HttpPanel.java]: Should be added the static modifier.

**Comment by kingthorin** [comment]
Will do, thanks

**Comment by kingthorin** [comment]
Done & done

**Comment by thc202** [comment]
Thank you.

**Pull request merged**